workflows/data_pipelines/rne/tmp/task_functions.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
+import re
+import logging
+from dag_datalake_sirene.helpers.minio_helpers import minio_client
+from dag_datalake_sirene.config import (
+    RNE_DB_TMP_FOLDER,
+    RNE_MINIO_FLUX_DATA_PATH,
+)
+
+
+def process_flux_json_files(**kwargs):
+    start_date = kwargs["ti"].xcom_pull(key="start_date", task_ids="get_start_date")
+
+    json_daily_flux_files = minio_client.get_files_from_prefix(
+        prefix=RNE_MINIO_FLUX_DATA_PATH,
+    )
+
+    if not json_daily_flux_files:
+        return None
+
+    if start_date is None:
+        start_date = "0000-00-00"
+
+    for file_path in sorted(json_daily_flux_files, reverse=False):
+        date_match = re.search(r"rne_flux_(\d{4}-\d{2}-\d{2})", file_path)
+        if date_match:
+            file_date = date_match.group(1)
+            if file_date >= start_date:
+                logging.info(f"Processing file {file_path} with date {file_date}")
+                minio_client.get_files(
+                    list_files=[
+                        {
+                            "source_path": RNE_MINIO_FLUX_DATA_PATH,
+                            "source_name": f"rne_flux_{file_date}.json",
+                            "dest_path": RNE_DB_TMP_FOLDER,
+                            "dest_name": f"{file_date}_rne_flux.json",
+                        }
+                    ],
+                )
+                json_path = f"{RNE_DB_TMP_FOLDER}{file_date}_rne_flux.json"
+                new_json_path = f"{RNE_DB_TMP_FOLDER}rne_flux_{file_date}.json"
+                with open(json_path, "r") as file:
+                    input_data = file.readlines()
+                output_data = []
```

Comment:
wont this fail on the big 60gb file ?

Reply:
Yes, it will. For now I created a copy of the file locally. And will either delete it and question API from scratch writing line by line or separate it into multiples files.